app/views/new-overview.html

```diff
@@ -355,6 +355,31 @@ <h2 ng-if="!app">
                   </div>
                 </div>
               </div>
+
+              <!-- service instances appear the same way no matter the filter listBy option -->
+              <div ng-if="overview.state.serviceInstances | hashSize">
+                <h2 ng-if="overview.state.serviceInstances">
+                  Provisioned Services
+                </h2>
+                <div class="list-pf">
+                  <!--
+                    todo: service-instance-row  :) its a special.
+                    todo: make deprovision work? may be just a delete....
+                    states: pending, failed to provision, deleting/deprovisioning...
+                    todo: try to pretty name:
+                          serviceClassDisplayName then
+                          serviceClassName then
+                          serviceInstanceName
+                          try to be pretty, if 2x+ of the same, will neeed the non-pretty
+                          name below... check specs and stuff.
+                  -->
+                  <service-instance-row
+                    ng-repeat="serviceInstance in overview.state.serviceInstances"
```

Comment:
we have started stable sorting most of our lists, i believe overview is sorted alphabetically @spadgett ?  You'll want to leave state.serviceInstances as is (my bind dialog is expecting it) and then have an orderedServiceInstances

You can rip off the code I used for the bind dialog to sort service instances based on their pretty name https://github.com/openshift/origin-web-console/pull/1395/files#diff-9b6377c25f28da45ed50c1d71ef2863aR60

you'll want to wait to sort (and show) them till both service classes and service instances are loaded, otherwise stuff will jump around